CRFModuleValidation/Assets.xcassets/Colors/dodgerBlue.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "colors" : [
+    {
+      "idiom" : "universal",
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "red" : "62",
```

Comment:
this is weird--some of the "srgb"-colorspace colors use integers from 0 to 255 for their red/green/blue components, some (e.g. macaroniAndCheese) use a float between 0.000 and 1.000. Do they all seem to work? If so, I guess the asset compiler decides what to do with them based on whether or not they have a decimal point?

Reply:
Not sure. I defined them all using 0-255.  The color name thing is iOS 11 only and I just used the Zeplin names for consistency.